### PR TITLE
Handle adding newlines for parsing ast -> string

### PR DESCRIPTION
**Pull request by patrickofriel-wk**
###### Overview:

Add logic to format query strings with newlines.

###### Reviewers:
@patrickofriel-wk
@dustinhiatt-wf
@chrisphillips-wk 
@davidgiametta-wk

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on HipChat: InfoSec Forum._

File: lib/sql.js [hunk 1]
```diff
@@ -165,13 +168,13 @@ function tablesToSQL(tables) {
 
     if (baseTable.db && baseTable.db !== null) str = baseTable.db + '.' + str;
     if (baseTable.as !== null) str += ' AS ' + identifierToSql(baseTable.as);
-
+
```
Comment (dustinhiatt-wf) [inline lib/sql.js]: Don't need this extra whitespace I don't think.

Comment (patrickofriel-wk) [reply lib/sql.js]: Updated

File: lib/sql.js [hunk 2]
```diff
@@ -181,9 +183,9 @@ function tablesToSQL(tables) {
         }
 
         if (joinExpr.as !== null) str += ' AS ' + identifierToSql(joinExpr.as);
-        if (has(joinExpr, 'on') && joinExpr.on !== null) str += ' ON ' + exprToSQL(joinExpr.on);
+        if (has(joinExpr, 'on') && joinExpr.on !== null) str += ' \nON ' + exprToSQL(joinExpr.on);
         if (has(joinExpr, 'using')) str += ' USING (' + joinExpr.using.map(identifierToSql).join(', ') + ')';
-
+
```
Comment (dustinhiatt-wf) [inline lib/sql.js]: Missed this; another one ;)

Comment (patrickofriel-wk) [reply lib/sql.js]: Updated again haha

**Comment by dustinhiatt-wf** [comment]
+1

**Pull request merged**